azext_iot/iothub/params.py

```diff
@@ -246,3 +246,88 @@ def load_iothub_arguments(self, _):
             options_list=["--content-type", "--ct"],
             help="MIME Type of file.",
         )
+
+    with self.argument_context("iot edge devices") as context:
+        context.argument(
+            "devices",
+            options_list=["--device", "-d"],
+            nargs="+",
+            action="append",
+            help="Space-separated key=value pairs corresponding to properties of the edge device to create. "
+            "The following key values are supported: `id`, `deployment` (inline json or path to file), `hostname`, "
+            "`parent`, `edge_agent`, and `container_auth` (inline json or path to file). "
+            "--device can be used 1 or more times. Review help examples for parameter usage."
+        )
+        context.argument(
+            "clean",
+            options_list=["--clean", "-c"],
+            arg_type=get_three_state_flag(),
+            help="Deletes all devices in target hub before creating new devices.",
+        )
+        context.argument(
+            "visualize",
+            options_list=["--visualize", "--vis", "-v"],
+            arg_type=get_three_state_flag(),
+            help="Shows visualizations of devices and progress of various tasks "
+            "(device creation, setting parents, updating configs, etc).",
+        )
+        context.argument(
+            "config_file",
+            options_list=["--config-file", "--config", "--cfg"],
+            help="Path to devices configuration file. Sample configuration file: "
+            "https://github.com/Azure/azure-iot-cli-extension/tree/dev/docs/samples/sample_devices_config.yaml",
+        )
+        context.argument(
+            "device_auth_type",
```

Comment:
can a person create a sym key device and a cert device in the same command? 

Not sure if we should be explicit here with "Device to hub authorization mechanism for all devices created in this command run" (will need to tweak wording)

Reply:
> can a person create a sym key device and a cert device in the same command?

Not in the same command, no. You can make one run of symmetric key devices, then another run for cert devices, etc - but only one auth type per `devices create` command.